openmls/src/credentials/mod.rs

```diff
@@ -273,6 +274,25 @@ impl Credential {
             MlsCredentialType::X509(cert) => cert.identity.as_slice(),
         }
     }
+
+    pub fn verify(&self) -> Result<(), CredentialError> {
+        match &self.credential {
+            MlsCredentialType::X509(cert) => {
+                // TODO: implement this on the whole chain with certval
```

Comment:
I don't think we'll be able to validate w/ certval from within OpenMLS; We need to to find a better design that allows us to defer to CC for this